### DISCUSSION TITLE
Strapi is good for Trello cloning?

### DISCUSSION THREAD

**Discussion by beactivesolutions** [strapi/strapi#6565] [Legacy Topics]
Is it an appropriate solution if you are responding to the project like Trello (boards and cart whit todo lists)?
It is expected to have 1 million records for the first 12 months (using PostgreSQL).

**Comment by robertpiosik** [comment]
Strapi doesn't have foreign key support **yet** so it will not provide your application's database with strict data integrity what would be necessary for such a big and frequently updated (in your use case) dataset. You also don't need data management strapi provides so I would suggest to go the RAW path and write the application from the ground up.

**Comment by beactivesolutions** [comment]
@robertpiosik - Does this mean the possibility of static content (like Contentful) or small projects (blog, small ecommerce, etc)? How reliable it is Strapi for big data project?

**Reply by robertpiosik** [reply]
I don't understand your question.

Strapi isn't meant to be used for big data project. It's a database schema design tool, with integrated editing dashboard and a layer for developing custom functionalities. I'm just trying to say that strapi doesn't ensure data integrity by not setting up foreign keys on tables yet. It might be a blocker in your use case.
